deps/llhttp/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.5.1)
 cmake_policy(SET CMP0069 NEW)
 
-project(llhttp VERSION )
+project(llhttp VERSION 6.0.9)
```

Comment:
There seem to be some spurious llhttp changes

Reply:
Yes, I'm rebasing this as we speak.